yaks-rails/lib/yaks-rails.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails'
+require 'yaks-html'
+
+module Rails
+  module Yaks
+    def self.global_config
+      @global_config ||= ::Yaks.new
+    end
+
+    class << self
+      attr_writer :global_config
+    end
+
+    def self.configure(&block)
+      @global_config = ::Yaks.new(&block)
+    end
+
+    module ControllerAdditions
+        def yaks(object, opts = {})
+          runner = Yaks.global_config.runner(object, {env: env}.merge(opts))
+          puts runner.media_type
+          render body: runner.call, content_type: runner.media_type
+        end
+
+      def self.included(base)
+        base.helper_method :yaks if base.respond_to? :helper_method
+      end
```

Comment:
`helper_method` makes the method available to the view. There's no use for calling `yaks` in your views, so no need for `helper_method` here.